$OpenBSD: patch-src_racket_src_gmp_gmplonglong_h,v 1.2 2014/07/15 00:13:16 juanfra Exp $

This patch changes some platforms to use the C code because these days
racket is mostly tested on amd64/i386 on other OS and this can help us in
the effort to port racket to other platforms.

--- src/racket/src/gmp/gmplonglong.h.orig	Thu Apr 17 21:58:42 2014
+++ src/racket/src/gmp/gmplonglong.h	Tue Jul 15 01:31:10 2014
@@ -96,6 +96,12 @@ MA 02111-1307, USA. */
    If any of these macros are left undefined for a particular CPU,
    C macros are used.  */
 
+/* These days only i386 and amd64 are tested */
+#if !defined(__i386__) || !defined(__x86_64__)
+# define MZ_GMP_NO_ASM 1
+# define NO_ASM 1
+#endif
+
 #ifndef MZ_GMP_NO_ASM
 
 /* The CPUs come in alphabetical order below.
